scalafmt-tests/src/test/resources/unit/Type.stat

```diff
@@ -85,3 +85,29 @@ type Row =
 type a = b_! # D
 >>>
 type a = b_! #D
+<<< with-chain types #1125
+type RequiredServices = Bag with
+  SomeOtherService.RequiredServices
+    with HasDynamicConfig with HasThrottles
+    with HasThrowableNotifier
+>>>
+type RequiredServices = Bag
+  with SomeOtherService.RequiredServices
+  with HasDynamicConfig
+  with HasThrottles
+  with HasThrowableNotifier
+<<< abstract def #1125
+trait NeedsServices {
```

Comment:
This probably shouldn't be in `Type.stat` since there is no `type`

Reply:
`TypeWith.stat` would be good, this is important enough to merit a separate suite.